WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -18,7 +18,21 @@ final class ShippingLabelAddressFormViewController: UIViewController {
     /// Top banner that shows a warning in case there is an error in the address validation.
     ///
     private lazy var topBannerView: TopBannerView = {
-        let topBanner = ShippingLabelAddressTopBannerFactory.addressErrorTopBannerView()
+        let topBanner = ShippingLabelAddressTopBannerFactory.addressErrorTopBannerView { [weak self] in
+            MapsHelper.openAppleMaps(address: self?.viewModel.address?.formattedPostalAddress) { [weak self] (result) in
+                switch result {
+                case .success:
+                    break
+                case .failure:
+                    self?.displayAppleMapsErrorNotice()
+                }
+            }
+        } contactCustomerPressed: { [weak self] in
+            if !PhoneHelper.callPhoneNumber(phone: self?.viewModel.address?.phone) {
```

Comment:
This seems to be failing for empty phone numbers

Reply:
Good catch, fixed here 991e3160aa4d8368fcb9f6b0b14e84a7a11fee5b